cirq/google/engine/engine.py

```diff
@@ -61,6 +62,20 @@ class ProtoVersion(enum.Enum):
 Program = Union[circuits.Circuit, Schedule]
 
 
+def user_project_header_request_builder(project_id):
```

Comment:
if we don't expected users to use this directly, make this "private" by leading "_"

Reply:
Should be private. Fixed